.gitignore

```diff
@@ -10,3 +10,4 @@ stack
 *~
 \#*\#
 .\#*
```

Comment:
These lines from when I initialized the project keep git from tracking any intermediate files (such as temporary backups) that my editor Emacs makes ;)